lib/shell/shellscript.js

```diff
@@ -50,8 +50,11 @@ class ShellScriptWorker {
 
         const preparedFiles = await prepareMetadata(rendition.directory);
 
+        // rendition options file for optiosn like postProcess
+        const optionsFile = path.resolve(rendition.directory, 'options.json');
```

Comment:
I think `prepareMetadata` is doing something similar. Have you considered encapsulating `options.json` inside it i.e
```
    return {
        errorFile: errFile,
        typeFile: typeFile
       optionsFile: optionsFile
    };
}
```